### ISSUE TITLE
Buttons shrink until they disappear on mouseover, making them un-clickable (UI scaling related)

### ISSUE THREAD

**Issue by C-Dub87** [VolantisDev/Launchpad#342]
When I move my cursor over a button to click it, it shrinks to the point that it disappears. This only happens when I have UI scaling switched on.

This makes Launchpad unusable on high-DPI displays.

I'm using Windows 11, 150% UI scale, 3840 x 2160 resolution.

**Comment by bmcclure** [comment]
Thanks for reporting this issue!

It seems like the GDI+ functionality for drawing GUI elements might need to be adjusted according to the current UI scale. I'll investigate this issue further and get a fix in place ASAP.

**Comment by bmcclure** [comment]
I have not had a chance to finish troubleshooting this issue yet, but I will get to it ASAP.

In the meantime, I'm curious if right-clicking Launchpad.exe, going to Properties, and enabling the Compatibility option to disable DPI scaling might make the application usable until it gets fixed?

**Comment by C-Dub87** [comment]
> I have not had a chance to finish troubleshooting this issue yet, but I will get to it ASAP.
> 
> In the meantime, I'm curious if right-clicking Launchpad.exe, going to Properties, and enabling the Compatibility option to disable DPI scaling might make the application usable until it gets fixed?

Yes, that does indeed resolve the scaling issue. Buttons are now clickable. Thanks!

**Comment by kingreboot** [comment]
I just found this project and would like to use it, but running into this same issue on windows 11. The option in the fix above does not seem to be there. Is there another fix?

**Comment by kingreboot** [comment]
Found the new renamed options and leaving here so if others find it they can do the same. Under compatibility there is a change high DPI settings button that opens a new window. Down at the bottom check override High DPI Scaling Behavior and set the drop down to System. That fixed my issue and I hope the author of this project relooks at this issue and fixes it.

**Comment by ReplicaObscura** [comment]
I'm glad you got it working for now! This does still need an official fix, although that might change soon.

I recently switched to an entirely new GUI for Launchpad which will ease most (hopefully all) of the usability and compatibility issues with the current version. It isn't ready yet, but it completely modernizes the interface, makes it responsive, and will ultimately allow it to work cross-platform as well. I'm very excited to officially unveil it soon.

I'll leave this open in the meantime in case others run into the same issue, and on the off chance that the new GUI is abandoned and I continue development of the current version.

**Comment by kingreboot** [comment]
well it fix that issue but the whole adding game feature was broken. I never got a chance to fully test what it could do. Also not sure if I should summit as its own issue.

**Comment by ReplicaObscura** [comment]
I've heard a few reports of people who get errors adding games, but it's hard to say what issue you're running into without further context. If you could submit a separate issue with as much detail as possible (what version of Launchpad, what version of Windows, what the error message is, and what steps to reproduce) I should hopefully be able to get that fixed.

It should also be fixed with the new release coming up, but since I don't have a date for that yet I'd love to get that figured out.

**Comment by kingreboot** [comment]
I will do this when I get some free time. Think the app is cool, just need time to mess with it.
